package.json

```diff
@@ -72,7 +72,7 @@
     "@hyperplay/chains": "^0.5.0",
     "@hyperplay/check-disk-space": "^3.5.2",
     "@hyperplay/quests-ui": "^0.1.27",
-    "@hyperplay/ui": "^1.11.1",
+    "@hyperplay/ui": "1.19.5-beta.0",
```

Comment:
would be good if you can wait my merge for the new alert cards so the new change reflects here.